wandb/sdk/artifacts/artifact.py

```diff
@@ -753,10 +783,20 @@ def _populate_after_save(self, artifact_id: str) -> None:
                     fileCount
                     createdAt
                     updatedAt
+                    ttlDurationSeconds
+                    ttlIsInherited
```

Comment:
Nit: keep the same order of attributes everywhere, this makes it easier e.g. to check if we set all the required attributes. This should go before `aliases`.